rpm/rpms.bzl

```diff
@@ -19,31 +19,53 @@ def rpms():
         name = "audit-libs-0__3.0.9-1.fc37.x86_64",
         sha256 = "23cda2a639c358757ee35ce6270ba3d0c6cd779309ff528e7c08c0239737dffb",
         urls = [
-            "https://mirror.netzwerge.de/fedora/linux/development/37/Everything/x86_64/os/Packages/a/audit-libs-3.0.9-1.fc37.x86_64.rpm",
-            "https://ftp.fau.de/fedora/linux/development/37/Everything/x86_64/os/Packages/a/audit-libs-3.0.9-1.fc37.x86_64.rpm",
-            "https://mirror.23m.com/fedora/linux/development/37/Everything/x86_64/os/Packages/a/audit-libs-3.0.9-1.fc37.x86_64.rpm",
-            "https://ftp.halifax.rwth-aachen.de/fedora/linux/development/37/Everything/x86_64/os/Packages/a/audit-libs-3.0.9-1.fc37.x86_64.rpm",
+            "https://ftp.plusline.net/fedora/linux/releases/37/Everything/x86_64/os/Packages/a/audit-libs-3.0.9-1.fc37.x86_64.rpm",
+            "https://mirror.23m.com/fedora/linux/releases/37/Everything/x86_64/os/Packages/a/audit-libs-3.0.9-1.fc37.x86_64.rpm",
+            "https://ftp.fau.de/fedora/linux/releases/37/Everything/x86_64/os/Packages/a/audit-libs-3.0.9-1.fc37.x86_64.rpm",
+            "https://mirror.netzwerge.de/fedora/linux/releases/37/Everything/x86_64/os/Packages/a/audit-libs-3.0.9-1.fc37.x86_64.rpm",
```

Comment:
Guess this becomes obsolete when we host the RPMs and become the sole mirror.
But in case we are not the sole mirror, is it possible to stabilize the ordering so that we don't end up with change of orderings being registered as changes? Makes it hard to review actual changes otherwise in the future.

Reply:
I think only using our archive or enforcing a stricter ordering are both good ideas. Will not be implemented here.